README.md

```diff
@@ -159,6 +159,11 @@ npm install glamorous-native --save
 
 You can learn more at the [glamorous-native project][glamorous-native].
 
+### Typescript
+
+`glamorous` includes typescript definition files.
+
+For usage notes and known issues see [typescript usage][typescript-usage].
```

Comment:
Maybe instead of "see typescript usage" it should be: "see `other/TYPESCRIPT_USAGE.md` so folks know where to find it if they want to navigate there themselves?